mne/viz/tests/test_topomap.py

```diff
@@ -109,6 +110,13 @@ def test_plot_topomap():
                     for x in subplot.get_children()
                     if isinstance(x, matplotlib.text.Text)))
 
+    # Plot array
+    for ch_type in ('mag', 'grad'):
+        evoked_ = evoked.copy().pick_types(eeg=False, meg=ch_type)
+        plot_topomap(evoked_.data[:, 0], evoked_.info)
```

Comment:
Does it also work with `evoked_.data[:, [0]]` which has shape `(n_channels, 1)` and `evoked_.data[:, :3]` which has multiple time points? It's not clear from the tests if those should work or not.